app/feature/feature-travel-certificate/src/main/graphql/QueryCurrentContracts.graphql

```diff
@@ -1,11 +1,7 @@
 query CurrentContracts {
   currentMember {
     activeContracts {
-      currentAgreement {
-        productVariant {
-          typeOfContract
-        }
-      }
+      supportsTravelCertificate
```

Comment:
Probably can also rename this file and query to show that we only are interested in this field now, not the rest of em